src/SSHDebugTests/PSOutputParserTests.cs

```diff
@@ -62,13 +64,14 @@ public void PSOutputParser_NoUserName()
         {
             // Made up ps output from a system where $USER wasn't a thing
             const string username = "";
+            const string architecture = "";
             const string input =
                 "pppppppppp rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr COMMAND\n" +
                 "         1 root                             /sbin/init\n" +
                 "       720                                  dbus-daemon --system --fork\n" +
                 "      2389 greggm                           -bash\n";
 
-            List<Process> r = PSOutputParser.Parse(input, username);
+            List<Process> r = PSOutputParser.Parse(input, new SystemInformation(username, architecture));
```

Comment:
>new SystemInformation(username, architecture) [](http://example.com/codeflow?start=58&length=45)

If I read the code correctly, `null` would actually be passed in this case.

Reply:
Oh, I see. In SSH, you have it right. In Pipe, it would be null. We should probably pick one.

Reply:
Added a suggestion for how to fix this.

Reply:
Kept it to be similar to `new SystemInformation(string.Empty, string.Empty);`